config/samples/catalogd_operatorcatalog.yaml

```diff
@@ -3,8 +3,9 @@ kind: ClusterCatalog
 metadata:
   name: operatorhubio
 spec:
+  priority: 0
   source:
     type: Image
     image:
+      pollIntervalMinutes: 1440
```

Comment:
Why are we changing the poll interval from 10 minutes to 1 day?

Reply:
There was conflicting values between operator-controller and catalogd here, in the case of the samples. I can make the sample 10 minutes if you prefer.

Reply:
I prefer 10m to 24h.

Reply:
Good catch. Thanks @joelanford